src/reporter/plugin-host.ts

```diff
@@ -193,8 +193,11 @@ export default class ReporterPluginHost {
         throw new Error('Not implemented');
     }
 
+    // NOTE: It's an optional method
+    public async reportInit (): Promise<void> {
+    }
```

Comment:
I don't feel that `reportInit` is the correct name for this method. `reportInit` means that it should report something about TestCafe's initialization state. Even the corresponding error `GFailed to initialize reporters`. So I we think we should name it just `init` and should consider adding the `dispose` method to make API symmetrical.